docker-storage-setup.sh

```diff
@@ -873,6 +987,82 @@ setup_storage() {
   fi
 }
 
+restore_selinux_context() {
+  local dir=$1
+
+  if ! restorecon -R -v $dir; then
+    echo "restorecon -R -v $dir failed."
+    return 1
+  fi
+}
+
+# Inherit the same selinux context as /var/lib/docker.
+setup_selinux_context_root_dir() {
+  local dir=$1
+  local policy
+
+  [ "$dir" == "/var/lib/docker" ] && return 0
+
+  # If there is already a labeling policy for the dir, don't try to add one.
+  policy=`semanage fcontext -l -C $dir | grep "^$dir =" | wc -l`
+  if [ $? -ne 0 ];then
+    echo "Querying existing selinux rules for $dir failed."
+    return 1
+  fi
+
+  # If no policy exists, add one.
+  if [ "$policy" == "0" ]; then
+    if ! semanage fcontext -a -e /var/lib/docker $dir; then
+      echo "semanage fcontext -a -e /var/lib/docker $dir failed."
+      return 1
+    fi
+  fi
+}
+
+get_docker_root_dir(){
+    local flag=false
+    options=$(grep -e "^OPTIONS" /etc/sysconfig/docker|cut -d"'" -f 2)
```

Comment:
Can we define path to docker file in a global variable instead of hardcoding `/etc/sysconfig/docker` here.